Resources/Prototypes/_Sunrise/Catalog/Fills/Crates/ussp.yml

```
@@ -53,13 +53,12 @@
   parent: CrateAmmunitionSmall
   id: CrateAmmunitionSmallLightRifle
   name: old small ammunition crate
+  suffix: Filled
   components:
   - type: StorageFill
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)